### ISSUE TITLE
form_agro1 Key Entry Form Inconsistency?

### ISSUE THREAD

**Issue by Patowhiz** [climsoft/Climsoft#683]
I noticed `form_agro1` design suggests that the elements are measured at specific time. However, when saving, the records are saved with the default hour 6 as part of the datetime. I wonder what the time labels **@ specifictimeGMT** stand for?

![formagromet](https://user-images.githubusercontent.com/21179007/156717768-8b3d695d-edfb-4645-8990-cc2e94b894bc.PNG)

**Comment by smachua** [comment]
@Patowhiz this a key entry form used only in Kenya to capture data used in making agro-met products in a monthly bulletin. The text boxes grouping labels are meant to guide the key entry operators since that how the actual form is designed. See the attachment.
The elements have been observed daily but at different hours. For convenience of their use, in the metadata they are defined with different element codes. For example soil temp at depths of 005cm @ 05Z, 09Z and 13Z have different element codes. This makes it easier for the agro-meteorologists to produce monthly summaries for each of these elements in one data extraction. Like any other daily observations they are archived with the default hour but we have been debating with KMD to harmonize with their actual observation hours or do away with this form and use the hourly form instead. We're yet to conclude this debate.

![9137048-FORM626-1984013106](https://user-images.githubusercontent.com/12034121/156737223-87be17d0-ded1-4ce5-8680-f7a609947512.jpg)

**Comment by Patowhiz** [comment]
@smachua thanks for your response. It now makes sense to me.

Just to make sure I understand you correctly, see my comments below;

So they have a paper form that has header labels with observation hour for specific elements (e.g 0500 GMT, 0900 GM, ), but they chose to archive the observations with hour 6 (based on the screenshot of the paper form sent you have provided above). So because of that it made sense for you to just have hour 6 as the default during saving for all the elements in this form (helps with fast data entry too).

And should they choose to use the correct observation hour as outlined in the paper form, then they can use the hourly form for archiving to climsoft.

@smachua If my comments above are correct. Then this issue can now be closed.

**Comment by smachua** [comment]
@Patowhiz you are right with those comments. Unless Marcellin has anything to say the issue can be closed.
